packages/components/src/RecurringSelect/DateCellCheckbox.css

```diff
@@ -0,0 +1,76 @@
+.checkboxWrapper {
+  position: relative;
+}
+
+/* kept present but hidden for accessibility */
+.dayCheckboxInput {
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+}
+
+.dayCheckbox {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: var(--space-smaller);
+  border-radius: var(--radius-large);
+  text-align: center;
+  user-select: none;
+  background: var(--color-surface);
+  transition: var(--timing-base);
+}
+
+@supports (aspect-ratio: 1/1) {
+  .dayCheckbox {
+    /*
+     * disabling property-no-unknown here as this is a real property
+     * that stylelint just doesn't know about. We may be able to remove
+     * this when we update our stylelint to not care about IE11.
+     */
+    /* stylelint-disable-next-line property-no-unknown */
+    aspect-ratio: 1/1;
```

Comment:
@chris-at-jobber @chriscarline i'm assuming we good with this bois being big bois when it's on a wide parent?
![image](https://user-images.githubusercontent.com/15986172/146281212-7404287e-7ad1-47dc-a86b-aba0f0870120.png)

same goes for the month
![image](https://user-images.githubusercontent.com/15986172/146281147-c389a368-1557-4f30-9ecb-6157d9003daa.png)

Reply:
chris said yes